doc/api/next_api_changes/removals/30005-DS.rst

```diff
@@ -0,0 +1,7 @@
+``matplotlib.cm.get_cmap``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have the name of a colormap as a `str`, use ``matplotlib.colormaps[name]`` instead.
+
+Otherwise `matplotlib.cm.ColormapRegistry.get_cmap` if you have a `str`, `None` or a
+`matplotlib.colors.Colormap` object that you want to convert to a `.Colormap` object.
```

Comment:
We should be a bit more verbose here and give a full overview of the new colormap API.

```suggestion
Colormaps are now available through the `.Coloarmap.Registry` accessible via
``matplotlib.colormaps`` or ``plt.colormaps``.

If you have the name of a colormap as a sting, you can use direct lookup
``matplotlib.colormaps[name]``. Alternatively, `matplotlib.colormaps.get_cmap` will
maintain the existing behavior of additionally passing through `.Colormap` instances
and converting ``None`` to the default colormap. `plt.get_cmap` will stay as a
shortcut to `matplotlib.colormaps.get_cmap`.
```